pkgs/os-specific/linux/syslinux/default.nix

```diff
@@ -65,19 +66,21 @@ stdenv.mkDerivation {
       "0018-prevent-pow-optimization.patch"
       "26f0e7b2"
       "sha256-dVzXBi/oSV9vYgU85mRFHBKuZdup+1x1BipJX74ED7E=")
+    (fetchpatch {
+      # Fixes build with "modern" gnu-efi
+      # https://github.com/samueldr/syslinux/commit/68defee52f4eba82eefaeea17f21c7498448dd6b
+      url = "https://github.com/samueldr/syslinux/commit/68defee52f4eba82eefaeea17f21c7498448dd6b.patch";
+      hash = "sha256-5xIqM8Gq8D86HL1i7fBTHRB/ZR2XtCz5VsT4bI5vneo=";
+    })
```

Comment:
tbh we would have been probably better off to just vendor that tiny patch to not rely on your repo 😅